src/sass/app/components/market.scss

```diff
@@ -288,6 +288,23 @@
                         font-size: 20px;
                         padding: 16px 0px;
                         font-weight: 700;
+                        flex: 1 0 auto;
+                        color: $COLOR_BLUE;
+                        transition: none;
+
+                        &.sticky {
+                            position: absolute;
+                            width: 431px;
+                            top: 60px;
+                            background: $COLOR_WHITE;
+                            @media (max-width: 767px) {
```

Comment:
why do we specify for 767px? could it cover for most mobile screens?

Reply:
The max width of the mobile view is 767px without it this will break the app.